lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -1714,17 +1714,27 @@ def cmd_db_status(*args)
   end
 
   def cmd_db_connect_help
-    print_line("   Usage: db_connect <user:pass>@<host:port>/<database>")
-    print_line("      OR: db_connect -y [path/to/database.yml]")
-    print_line("      OR: db_connect [options] <http|https>://<host:port>")
-    print_line("Examples:")
-    print_line("       db_connect user@metasploit3")
-    print_line("       db_connect user:pass@192.168.0.2/metasploit3")
-    print_line("       db_connect user:pass@192.168.0.2:1500/metasploit3")
-    print_line("       db_connect http://localhost:8080")
-    print_line("       db_connect -c ~/cert.pem -t 6a7a74c1a5003802c955ead1bbddd4ab1b05a7f2940b4732d34bfc555bc6e1c5d7611a497b29e8f0 https://localhost:8080")
-    print_line("       db_connect --name LA-server http://laoffice.org:8080")
-    print_line("       db_connect LA-server")
+    print_line("   USAGE:")
+    print_line("      * Postgres Data Service:")
+    print_line("          db_connect <user:[pass]>@<host:[port]>/<database>")
+    print_line("        Examples:")
+    print_line("          db_connect user@metasploit3")
+    print_line("          db_connect user:pass@192.168.0.2/metasploit3")
+    print_line("          db_connect user:pass@192.168.0.2:1500/metasploit3")
+    print_line("          db_connect -y [path/to/database.yml]")
+    print_line(" ")
+    print_line("      * HTTP Data Service:")
+    print_line("          db_connect [options] <http|https>://<host:port>")
```

Comment:
Should `port` here be optional and default when to `80/443` based on `http/https`?  This would also suggest  another example.

```
db_connect https://my-super-msf-data.service.com
```